### PR TITLE
Component highlight improvements

### PR DESCRIPTION
**Pull request by andrecardoso**
## Description
- Fix component highlight cut off at the bottom in fullscreen image template
- Increase zindex value of component highlight to fix highlight not appearing in Google Slides and Simple Weather template
- Remove highlight when highlight div is clicked 

## Motivation and Context

Fix issues with component highlight

## How Has This Been Tested?
Tested locally and added unit tests

## Release Plan:
- As the Submitter, upon requesting review of this pull request, I confirm that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed. 
- As the Reviewer, upon approving the changes in this PR, I confirm I have reviewed and I agree that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed

#### Release Checklist Items Skipped?
If any Release Checklist items were intentionally skipped, please provide which ones and the reasons why

### CODE REVIEW

File: src/rise-preview.js [hunk 1]
```diff
@@ -60,11 +60,20 @@ RisePlayerConfiguration.Preview = (() => {
 
     if ( el ) {
       const rect = el.getBoundingClientRect();
+      const bodyRect = document.body.getBoundingClientRect();
+
+      const elementWidth = rect.right - rect.left;
+      const bodyWidth = bodyRect.right - bodyRect.left;
+      const highlightWidth = elementWidth > bodyWidth ? bodyWidth : elementWidth;
```
Comment (olegrise) [inline src/rise-preview.js]: @andrecardoso would it be simpler to just pick the smallest `right` value. Example:
```
const minRight = Math.min(rect.right, bodyRect.right);
```
and then below do this:
```
divHighlight.style.width = ( minRight - rect.left ) + "px";
```

Comment (andrecardoso) [reply src/rise-preview.js]: I thought about doing this, but then there is the risk of using the body width and the element's height when only the width is larger. I don't know if that's really an issue though. Let me try it.

Comment (andrecardoso) [reply src/rise-preview.js]: It worked perfectly. It even fixed that problem where the background image goes to the right of the template. I'll commit the change.

**Comment by andrecardoso** [comment]
@olegrise please re-review. Thanks

**Pull request merged**